src/main/webapp/WEB_INF/view/index.jsp

```diff
@@ -0,0 +1,9 @@
+<%@ page contentType="text/html;charset=UTF-8"%>
+<html>
+    <head>
+        <title>index</title>
+    </head>
+    <body>
+        <h1>Information written to our resource</h1>
+    </body>
```

Comment:
"Create a JSP page that will display "Information written to our resource" when we write to the resource"

Reply:
done